Servers/UI/OJS.Servers.Ui.Models/Contests/ContestSortingRequestModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace OJS.Servers.Ui.Models.Contests;
+
+public class ContestSortingRequestModel
+{
+    [BindProperty(Name = "sorttype")]
```

Comment:
Is this necessary?

Reply:
I don't use this class, but forgot to delete it. It is deleted now.